Content.Server.Database/Model.cs

```diff
@@ -1119,4 +1120,16 @@ public class RoleWhitelist
         [Required]
         public string RoleId { get; set; } = default!;
     }
+
+    [Table("text_automod")]
+    public sealed class TextAutomod
+    {
+        [Column("id")] public int Id { get; set; }
```

Comment:
You don't need to specify this name explicitly anymore.

Reply:
Removed, but it changed the migration. [1fcd320](https://github.com/space-wizards/space-station-14/pull/28570/commits/1fcd320ced818ca06a31d6cf688fec7a55e41b6f)